db.js

```diff
@@ -365,16 +361,11 @@ exports.init = function (sbot, config) {
         } else if (SSBURI.isBendyButtV1FeedSSBURI(msgVal.author)) {
           const previous = (state[msgVal.author] || { value: null }).value
           const err = bendyButt.validateSingle(msgVal, previous, hmacKey)
-          if (err) {
-            debug(
-              `validation failed for bendy butt message in addImmediately(): ${err.message}`
-            )
-            return cb(err)
-          }
```

Comment:
With the new changes, I was getting a bunch of lines formatted like this under `prettier`, and it was getting hard to follow the "normal" (non-error) control flow, so I decided to turn off prettier for these lines that handle errors with clarify-error

Reply:
I like the shorter lines here. And I guess not doing debug is fine, as we'll catch these errors further up the stack.